src/Compiler/Checking/FindUnsolved.fs

```diff
@@ -20,275 +20,275 @@ type env = | NoEnv
 let FindUnsolvedStackGuardDepth = StackGuard.GetDepthOption "FindUnsolved"
 
 /// The environment and collector
-type cenv = 
-    { g: TcGlobals 
-      amap: Import.ImportMap 
-      denv: DisplayEnv 
-      mutable unsolved: Typars 
+type cenv =
+    { g: TcGlobals
+      amap: Import.ImportMap
+      denv: DisplayEnv
+      mutable unsolved: Typars
       stackGuard: StackGuard }
 
     override _.ToString() = "<cenv>"
 
 /// Walk types, collecting type variables
-let accTy cenv _env (fallbackRange: Range option) ty =
+let accTy cenv _env (fallbackRange: Range) ty =
```

Comment:
Nit: the convention is to use `range` instead of `Range`.
And it might also be slightly more consistent to go with `mFallback` as `m` is used as an abbreviation for `range` https://github.com/dotnet/fsharp/blob/main/docs/coding-standards.md#abbreviations